docker/kyuubi-service.yaml

```diff
@@ -0,0 +1,41 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Usage:
+# Start:
+#   kubectl apply -f https://raw.githubusercontent.com/apache/incubator-kyuubi/master/docker/kyuubi-service.yaml
+# Connect:
+#   ${SPARK_HOME}/bin/beeline -u 'jdbc:hive2://${hostname}:30009'
```

Comment:
What would the `hostname` be?

Reply:
ip/hostname of any k8s host.

Reply:
Can it be static? something like kyuubi.apache.org:30009

Reply:
It's host that your k8s deployed(any host of k8s cluster), not pod or service. So, the name is related to the user's own definition.

Reply:
if they are not able to be used directly, these yml files shall be suffixed with `template` or `example`

Reply:
Agree, but i think it can use directly.
We can do as usage:
1. create pod
```
[user@kyuubi-test kyuubi]# kubectl apply -f https://raw.githubusercontent.com/apache/incubator-kyuubi/master/docker/kyuubi-pod.yaml
```
2. create NodePort
```
[user@kyuubi-test kyuubi]# kubectl apply -f https://raw.githubusercontent.com/apache/incubator-kyuubi/master/docker/kyuubi-servie.yaml`
```
3. connect to kyuubi on any host that can connect to host `kyuubi-test`
```
[user@any spark]# bin/beeline -u 'jdbc:hive2://kyuubi-test:30009'
```
It is also appropriate to make these two files suffixed with template or example.

Reply:
Do you mean the `kyuubi-test` here is static for the service, or even the pod is evicted or crashed

Reply:
Expose a node port means that expose the port to all K8s nodes. Suppose the K8s deployed in 3 hosts `k8s-1`, `k8s-2`, `k8s-3`, any of `jdbc:hive2://k8s-1:30009`, `jdbc:hive2://k8s-2:30009`, `jdbc:hive2://k8s-3:30009` works.
The static `kyuubi-test` is misleading to me.

Reply:
how about `${hostname}` => `${any_hostname_of_k8s_nodes}`?

Reply:
sure, `any_hostname_of_k8s_nodes` is better here.